README.md

```diff
@@ -125,6 +125,9 @@ Some errors shouldn't cause your stoplight to move into the red state. Usually
 these are handled elsewhere in your stack and don't represent real failures. A
 good example is `ActiveRecord::RecordNotFound`.
 
+By default all errors that inherit from StandardError will be caught by stoplight. 
```

Comment:
Nits: "StandardError" should be in backticks, and "by stoplight" can be removed.